src/payment/strategies/googlepay/googlepay-adyenv3-initializer.ts

```diff
@@ -0,0 +1,111 @@
+import { round } from 'lodash';
+
+import { Checkout } from '../../../checkout';
+import PaymentMethod from '../../payment-method';
+
+import { BillingAddressFormat, GooglePaymentData, GooglePayInitializer, GooglePayPaymentDataRequestV2, TokenizePayload } from './googlepay';
+
+export default class GooglePayAdyenV3Initializer implements GooglePayInitializer {
+    initialize(
+        checkout: Checkout,
+        paymentMethod: PaymentMethod,
+        hasShippingAddress: boolean
+    ): Promise<GooglePayPaymentDataRequestV2> {
```

Comment:
Is it still supposed to be V2?

Reply:
Yes, it's Google Pay api version, not the Adyen